libraries/botbuilder-dialogs/src/prompts/oauthPrompt.ts

```diff
@@ -124,6 +124,7 @@ export interface OAuthPromptSettings {
  * ```
  */
 export class OAuthPrompt extends Dialog {
+    private readonly PersistedCaller: string = 'botbuilder-dialogs.caller';
```

Comment:
This made me remember that this didn't get done in c#, as somebody merged my PR overnight. Don't think it's in python either. I'll tlak to tracy and do my best to remember to update c#

Reply:
I was just looking at @tracyboehrer's [PR](https://github.com/microsoft/botbuilder-python/pull/1014/files) I'm not sure if we need to do this here... This key may be stored one level higher than where developers interact.

Reply:
@stevenic or @johnataylor could you chime in?